tez-api/src/main/java/org/apache/tez/runtime/api/events/InputDataInformationEvent.java

```diff
@@ -103,6 +113,7 @@ public Object getDeserializedUserPayload() {
   public String toString() {
     return "InputDataInformationEvent [sourceIndex=" + sourceIndex + ", targetIndex="
         + targetIndex + ", serializedUserPayloadExists=" + (userPayload != null)
-        + ", deserializedUserPayloadExists=" + (userPayloadObject != null) + "]";
-  } 
+        + ", deserializedUserPayloadExists=" + (userPayloadObject != null)
+        + ", serializedPath=" + serializedPath + "]";
```

Comment:
If serializedPath is null then it would look a bit odd. I like to handle this type of scenario. But this is just styling :)

Reply:
I can add a null check like:
```
 + serializedPath != null ? (", serializedPath=" + serializedPath) : "" + "]";
```